### PR TITLE
Add short doc on Config Sync to the kpt.dev site

### PR DESCRIPTION
**Pull request by mortent**

### CODE REVIEW

File: site/gitops/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Gitops
+
+Packages eventually have to be deployed into one or more clusters. The default
+deployment mechanism is [Config Sync](gitops/configsync/), but since configuration is stored in standard
+repositories, other gitops tools can also be used.
```
Comment (bgrant0607) [inline site/gitops/README.md:5]: GitOps (capitalization) -- here and everywhere

I also suggest linking to https://opengitops.dev/ on the first mention in the body of the text

File: site/sidebar.md [hunk 2]
```diff
@@ -37,6 +37,9 @@
 	- [8 Package Orchestration](book/08-package-orchestration/)
 		- [8.1 System Requirements](book/08-package-orchestration/01-system-requirements.md)
 		- [8.2 Quickstart](book/08-package-orchestration/02-quickstart.md)
+		- [8.3 Registering a Repository](book/08-package-orchestration/03-registering-a-repository.md)
```
Comment (bgrant0607) [inline site/sidebar.md:40]: Are these changes related to this PR?

File: site/sidebar.md [hunk 3]
```diff
@@ -90,6 +93,8 @@
 - [Functions Catalog](https://catalog.kpt.dev/ ":target=_self")
   - [Curated](https://catalog.kpt.dev/ ":target=_self")
   - [Contrib](https://catalog.kpt.dev/contrib/ ":target=_self")
+- [Gitops](gitops/)
```
Comment (bgrant0607) [inline site/sidebar.md:96]: GitOps

File: site/gitops/configsync/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+Config Sync also lets development teams independently manage their
+namespaces within clusters, while still being subject to policy guardrails set
+by administrators.
+
+Config Sync is an open-source project and the source and releases available at
+[github.com/GoogleContainerTools/kpt-config-sync](https://github.com/GoogleContainerTools/kpt-config-sync).
+It also provides instructions for installing Config Sync on your cluster and
+building it from source.
+
+Config Sync is also available with [Anthos Config Management (ACM)](https://cloud.google.com/anthos/config-management)
```
Comment (bgrant0607) [inline site/gitops/configsync/README.md:17]: I suggest linking to the Config Sync overview.

**Review by bgrant0607 — COMMENTED** [review]
Thanks!

File: site/sidebar.md (hunk 2 above)
Comment (mortent) [reply site/sidebar.md:40]: No, I think this is just from a previous PR that didn't run the script to generate the sidebar. I thought I'd just keep it here to get it fixed, but I can move it into a separate PR if you prefer that.

**Review by droot — APPROVED** [review]

**Review by sdowell — APPROVED** [review]
